src/app/notifications/Notification.scss

```diff
@@ -48,6 +48,26 @@
     margin-bottom: 4px;
   }
 
+  .undo {
+    margin: 10px auto 0px;
+    padding: 6px;
+    border-radius: 4px;
+    text-align: center;
+    color: #888;
+    background-color: #222;
+    display: block;
+    width: 30%;
+    cursor: pointer;
+    &::before {
+      content: '\21a9\fe0f';
```

Comment:
Let's use the `AppIcon` component rather than `::before` styling - these unicode symbols can get mapped to weird things in different fonts.